src/audio/google/google_rtc_audio_processing.c

```diff
@@ -333,6 +336,7 @@ static int google_rtc_audio_processing_init(struct processing_module *mod)
 
 	cd->config.reference_fmt = reference_fmt.audio_fmt;
 	cd->config.output_fmt = output_fmt.audio_fmt;
+	cd->config = *(const struct sof_ipc4_aec_config *)cfg->init_data;
```

Comment:
this overwrites all previous assignments of individual `cd->config.*` fields, can they all be removed?

Reply:
incorrect merge, to be fixed